src/tools/imgtool/iflopimg.cpp

```diff
@@ -45,29 +45,29 @@ imgtoolerr_t imgtool_floppy_error(floperr_t err)
 
 static void imgtool_floppy_closeproc(void *file)
 {
-	stream_close((imgtool_stream *) file);
+	delete (imgtool::stream *) file;
```

Comment:
Could you please make these use `reinterpret_cast` to make it clear what kind of cast you're attempting.